plutus-core/cost-model/budgeting-bench/Bench.hs

```diff
@@ -320,7 +320,7 @@ benchNop3 gen =
    DivideInteger:     RemainderInteger, QuotientInteger, ModInteger
    LessThanInteger:   GreaterThanInteger
    LessThanEqInteger: GreaterThanEqInteger
-   LtByteString:      GtByteString
+   LessThanByteString:      GreaterThanByteString
```

Comment:
Do we care about the ruined formatting here?

Reply:
I don't :)

Reply:
I'll probably notice this and fix it the next time I'm working on this file.